py/picca/blinding_a.py

```diff
@@ -1,71 +1,112 @@
 """This module defines some functions to blind data using strategy A"""
 import numpy as np
-
 from picca.constants import Cosmo, ABSORBER_IGM
+import sys, os  # requiered to mute Cosmo and dont print the
+                #   cosmology change on screen
 
-# Resample flux function from desispec/interpolation:
-# def resample_flux(xout, x, flux, ivar=None, extrapolate=False):
 
+# disable print to null to avout cosmo function outputs with O_m values
+def blockPrint():
+    """Redirect stdout to null (mute print to screen functions)
+        until enablePrint() is called
 
-def calcMaps(scale=1.0, Om=0.315):
-    """initialize class and compute the conversion maps
+    """
+    sys.stdout = open(os.devnull, 'w')
 
-    Arguments:
-    args: float - default: 1.0
-    Scaling of ommega matter with respect to the fiducial cosmology
+# Restore
+def enablePrint():
+    """Redirect stdout to __stdout__ (Unmute print to screen functions)
+        after enablePrint() is called
 
-    Om: float - default: 0.315
-    Omega matter in the fiducial cosmology
     """
-   ###### Definition of the fiducial cosmological model
+    sys.stdout = sys.__stdout__
 
-   fid_Om = Om
-   fid_Or = 0
-   fid_Ok = 0
-   fid_wl = -1
-
-   cosmo_m = Cosmo(Om=fid_Om,Or=fid_Or,
-                   Ok=fid_Ok,wl=fid_wl, blinding=False)
-   lambda_abs = ABSORBER_IGM['LYA']
+def calcMaps(scale=.95, Om=0.315):
+    """Initialize class and compute the conversion maps
 
-   ###### Definition of blind cosmological model
+    Args:
+        scale: float - default: .95
+            Scaling of ommega matter with respect to the fiducial cosmology
+        Om: float - default: 0.315
+            Omega matter in the fiducial cosmology
+    Returns:
+        z_grid, z_map: input and output values of mapping functions
+            to interpolate and blind the qso redshift value: z' = z+zmap(z)
+        l_grid, l_map: input and output values of mapping functions
```

Comment:
I would call them lambda_grid, lambda_map

Reply:
Changed it in picca__deltas.py and py/picca/blinding_a.py.